Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -272,17 +264,22 @@ private function whenRootNodeAggregateDimensionsWereUpdated(RootNodeAggregateDim
         }
 
         // delete all hierarchy edges of the root node
-        $this->getDatabaseConnection()->executeUpdate('
-                DELETE FROM ' . $this->tableNames->hierarchyRelation() . '
-                WHERE
-                    parentnodeanchor = :parentNodeAnchor
-                    AND childnodeanchor = :childNodeAnchor
-                    AND contentstreamid = :contentStreamId
-            ', [
-            'parentNodeAnchor' => NodeRelationAnchorPoint::forRootEdge()->value,
-            'childNodeAnchor' => $rootNodeAnchorPoint->value,
-            'contentStreamId' => $event->contentStreamId->value,
-        ]);
+        $deleteHierarchyRelationsStatement = <<<SQL
+            DELETE FROM {$this->tableNames->hierarchyRelation()}
```

Comment:
here doc + php inline string substitution is _not_ my friend because i know many things wont work here. But as simple method calls _do_ work we can really use it.
Or are there any votes against it? `FROM ' . $this->tableNames->hierarchyRelation() . '` might have better editor support for people not using PHPStorm but ...

Reply:
> here doc + php inline string substitution is not my friend because i know many things wont work here

What things do you mean and are they relevant in these places?

I also think that we wouldn't need it for one-liners, but I went for consistency here. And it looks much nicer in the IDE IMO

Reply:
I replaced a couple of more occurrences of string-based SQL queries.
So now in the `Neos.ContentGraph.DoctrineDbalAdapter` SQL queries are only built via the QueryBuilder or with Heredoc